core/src/stages/move_to.cpp

```diff
@@ -120,6 +120,16 @@ bool MoveTo::getJointStateGoal(const boost::any& goal, const moveit::core::Joint
 	} catch (const boost::bad_any_cast&) {
 	}
 
+	try {
+		const std::map<std::string, double>& joint_map = boost::any_cast<std::map<std::string, double>>(goal);
+		for (const auto& joint : joint_map) {
+			state.setVariablePosition(joint.first, joint.second);
```

Comment:
Should we check for the validity of the joint name, i.e. whether it is part of JMG?